tools/api-markdown-documenter/src/test/snapshots/html/simple-suite-test/sparse-config/simple-suite-test/testclass-testclassgetterproperty-property.html

```diff
@@ -21,6 +21,10 @@ <h3 id="testclassgetterproperty-signature">
           /** @virtual */
           <br>
           get testClassGetterProperty(): number;
+          <br>
+          <br>
+          <br>
+          set testClassGetterProperty(newValue: number);
```

Comment:
3 brs expected? I also see empty tds in a table in a file further down.

Reply:
Yeah, I'm not sure why 3 breaks are being generated here. At some point I want to rethink how line breaks are handled by the system (it currently does some custom things I don't think it needs to be doing). But it's unrelated to the changes at hand here, and shouldn't be a problem.

The empty `<td>`s below are expected, though. The tables reserve columns for extra metadata when any is present _anywhere_ in the table. But for the row items that don't have any such metadata, the cells will be empty.